event_log_profiling/src/main/scala/org/apache/spark/sql/rapids/tool/profiling/ProfileMain.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.rapids.tool.profiling
+
+import scala.collection.mutable.ArrayBuffer
+
+/**
+ * A profiling tool to parse Spark Event Log
+ * This is the Main function.
+ */
+
+object ProfileMain {
+  def main(args: Array[String]) {
+
+    // This tool's output log file name
+    val logFileName = "event_log_profiling.log"
+
+    // Parsing args
+    val appArgs = new ProfileArgs(args)
+    val eventlogPaths = appArgs.eventlog()
+    val outputDirectory = appArgs.outputDirectory().stripSuffix("/")
+
+    // Create the same logger and sparkSession used for ALL Applications.
+    val logger = ProfileUtils.createLogger(outputDirectory, logFileName)
+    val sparkSession = ProfileUtils.createSparkSession
```

Comment:
I didn't realize we are creating a spark session for this, I thought we were just using the classes, this essentially requires spark then to run, I need to think about that

Reply:
We were creating uber jar for this reason and then run using java command to run. Let us know your inputs on handling this in a better way.